frontend/src/v5/ui/controls/chip/filterChip/filterChip.styles.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled, { css } from 'styled-components';
+import { Chip } from '../chip.component';
+
+export const FilterChip = styled(Chip).attrs(({ theme }) => ({
+	color: theme.palette.base.main,
+	clickable: false,
+}))<{ selected?: boolean }>`
+	${({ theme }) => theme.typography.kicker};
+	cursor: pointer;
+	height: 18px;
+	border-radius: 5px;
+	padding: 3px 7px;
+	margin: 0;
+	font-size: 8px;
+	letter-spacing: 0.3px;
+	border: 1px solid;
+	.MuiChip-label {
+		padding: 0;
+	}
+	:hover {
```

Comment:
Not a change request, just a heads up to be sure this is intended. I saw you sometimes write
```scss
div {
     :hover { ... }
}
// instead of
div {
    &:hover { ... }
}
```
The first selector does not target the direct element itself, rather, all the descendants

![image](https://user-images.githubusercontent.com/43547703/221243822-2bcdd769-1ccb-4d16-b2dd-9a73171fdcf6.png)

In most cases, it should fine, but unless that is intended, I think it's better to avoid this as it could lead to subtle UI errors

Reply:
Thanks, very good to know